### PR TITLE
[FIX] l10n_it: translation

### PR DESCRIPTION
**Pull request by malb-odoo**
Ideally, we develop in English any localization (then translate to the domestic language, in this case Italian) All the module has been translated in english and then a .po file has been create to translate back in Italian

Task-id: 3082347

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_it/data/account_tax_template.xml [hunk 1]
```diff
@@ -140,7 +140,7 @@
     </record>
     <!-- Purchase of Goods (Acquisti Merce) ................................................... -->
     <record id="22am" model="account.tax.template">
-        <field name="description">22am</field>
+        <field name="description">22PG</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_template.xml:143]:
```suggestion
        <field name="description">22%</field>
```
The description should only mention the rate (seen with TSB).
--> Several occurrences.

File: addons/l10n_it/data/account.fiscal.position.template.csv [hunk 2]
```diff
@@ -1,5 +1,5 @@
 "name","chart_template_id:id","id","sequence","auto_apply","vat_required","country_id:id","country_group_id:id","note"
-"Italia","l10n_it_chart_template_generic","it",1,1,1,base.it,,
-"Regime Extra comunitario","l10n_it_chart_template_generic","extra",4,1,,,,
-"Regime Intra comunitario privato","l10n_it_chart_template_generic","intra_private",2,1,,,base.europe,
-"Regime Intra comunitario","l10n_it_chart_template_generic","intra",3,1,1,,base.europe,"Fattura emessa ai sensi dell’art. 17, comma 2 del DPR 26/10/1972 n. 633, l’applicazione dell’IVA è a carico del destinatario."
+"Italy","l10n_it_chart_template_generic","it",1,1,1,base.it,,
+"Extra-Community Regime","l10n_it_chart_template_generic","extra",4,1,,,,
+"Regime Intra-community private","l10n_it_chart_template_generic","intra_private",2,1,,,base.europe,
+"Regime Intra-community","l10n_it_chart_template_generic","intra",3,1,1,,base.europe,"Invoice issued in accordance with Article 17, Paragraph 2 of Presidential Decree No. 633 dated October 26, 1972, the application of VAT is the responsibility of the recipient."
```
Comment (epictete) [inline addons/l10n_it/data/account.fiscal.position.template.csv:5]:
```suggestion
"Domestic","l10n_it_chart_template_generic","it",1,1,1,base.it,,
"Extra-EU","l10n_it_chart_template_generic","extra",4,1,,,,
"Extra-EU (Private)","l10n_it_chart_template_generic","intra_private",2,1,,,base.europe,
"Intra-EU","l10n_it_chart_template_generic","intra",3,1,1,,base.europe,"Invoice issued in accordance with Article 17, Paragraph 2 of Presidential Decree No. 633 dated October 26, 1972, the application of VAT is the responsibility of the recipient."
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 3]
```diff
@@ -192,7 +192,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vp12" model="account.report.line">
-                        <field name="name">VP12 - Interessi dovuti per liquidazioni trimestrali</field>
+                        <field name="name">VP12 - Interest due for quarterly settlements.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:195]:
```suggestion
                        <field name="name">VP12 - Interest due for quarterly settlements</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 4]
```diff
@@ -294,7 +294,7 @@
                 <field name="code">VJ</field>
                 <field name="children_ids">
                     <record id="tax_report_line_vj1" model="account.report.line">
-                        <field name="name">VJ1 - Acquisti di beni dalla Città del Vaticano e da San Marino</field>
+                        <field name="name">VJ1 - Purchases of goods from Vatican City and San Marino.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:297]:
```suggestion
                        <field name="name">VJ1 - Purchases of goods from Vatican City and San Marino</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 5]
```diff
@@ -305,7 +305,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj2" model="account.report.line">
-                        <field name="name">VJ2 - Estrazione di beni da depositi Iva</field>
+                        <field name="name">VJ2 - Extraction of goods from VAT warehouses.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:308]:
```suggestion
                        <field name="name">VJ2 - Extraction of goods from VAT warehouses</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 6]
```diff
@@ -327,7 +327,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj4" model="account.report.line">
-                        <field name="name">VJ4 - Compensi corrisposti ai rivenditori di biglietti di viaggio ed ai rivenditori di documenti di sosta </field>
+                        <field name="name">VJ4 - Fees paid to resellers of travel tickets and resellers of parking documents.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:330]:
```suggestion
                        <field name="name">VJ4 - Fees paid to resellers of travel tickets and resellers of parking documents</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 7]
```diff
@@ -349,7 +349,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj6" model="account.report.line">
-                        <field name="name">VJ6 - Acquisti di rottami e altri materiali di recupero</field>
+                        <field name="name">VJ6 - Purchases of scrap and other recovered materials.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:352]:
```suggestion
                        <field name="name">VJ6 - Purchases of scrap and other recovered materials</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 8]
```diff
@@ -360,7 +360,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj7" model="account.report.line">
-                        <field name="name">VJ7 - Acquisti di oro industriale e argento puro effettuati in Italia</field>
+                        <field name="name">VJ7 - Purchases of industrial gold and pure silver made in Italy.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:363]:
```suggestion
                        <field name="name">VJ7 - Purchases of industrial gold and pure silver made in Italy</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 9]
```diff
@@ -371,7 +371,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj8" model="account.report.line">
-                        <field name="name">VJ8 - Acquisti di oro da investimento effettuati in Italia</field>
+                        <field name="name">VJ8 - Investment gold purchases made in Italy.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:374]:
```suggestion
                        <field name="name">VJ8 - Investment gold purchases made in Italy</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 10]
```diff
@@ -393,7 +393,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj10" model="account.report.line">
-                        <field name="name">VJ10 - Importazioni di rottami e altri materiali di recupero</field>
+                        <field name="name">VJ10 - Imports of scrap and other recovered materials.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:396]:
```suggestion
                        <field name="name">VJ10 - Imports of scrap and other recovered materials</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 11]
```diff
@@ -404,7 +404,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj11" model="account.report.line">
-                        <field name="name">VJ11 - Importazioni di oro industriale e argento puro</field>
+                        <field name="name">VJ11 - Imports of industrial gold and pure silver.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:407]:
```suggestion
                        <field name="name">VJ11 - Imports of industrial gold and pure silver</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 12]
```diff
@@ -415,7 +415,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj12" model="account.report.line">
-                        <field name="name">VJ12 - Subappalto di servizi in campo edile</field>
+                        <field name="name">VJ12 - Subcontracting of services in the construction field.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:418]:
```suggestion
                        <field name="name">VJ12 - Subcontracting of services in the construction field</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 13]
```diff
@@ -426,7 +426,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj13" model="account.report.line">
-                        <field name="name">VJ13 - Acquisti di fabbricati o porzioni di fabbricati strumentali</field>
+                        <field name="name">VJ13 - Purchases of buildings or portions of buildings used for capital purposes.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:429]:
```suggestion
                        <field name="name">VJ13 - Purchases of buildings or portions of buildings used for capital purposes</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 14]
```diff
@@ -448,7 +448,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj15" model="account.report.line">
-                        <field name="name">VJ15 - Acquisti di prodotti elettronici</field>
+                        <field name="name">VJ15 - Purchases of electronic products.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:451]:
```suggestion
                        <field name="name">VJ15 - Purchases of electronic products</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 15]
```diff
@@ -459,7 +459,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj16" model="account.report.line">
-                        <field name="name">VJ16 - Prestazioni di servizi in campo edile</field>
+                        <field name="name">VJ16 - Provision of services in the construction field.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:462]:
```suggestion
                        <field name="name">VJ16 - Provision of services in the construction field</field>
```

File: addons/l10n_it/data/account_tax_report_data.xml [hunk 16]
```diff
@@ -470,7 +470,7 @@
                         </field>
                     </record>
                     <record id="tax_report_line_vj17" model="account.report.line">
-                        <field name="name">VJ17 - Acquiti di beni e servizi del settore energetico</field>
+                        <field name="name">VJ17 - Purchases of energy sector goods and services.</field>
```
Comment (epictete) [inline addons/l10n_it/data/account_tax_report_data.xml:473]:
```suggestion
                        <field name="name">VJ17 - Purchases of energy sector goods and services</field>
```

File: addons/l10n_it/data/account.account.template.csv [hunk 17]
```diff
@@ -1,185 +1,185 @@
...
+1506,1506,Change them at a discount,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1507,1507,Change in collection,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1508,1508,Accounts receivable from customers (PoS),asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1509,1509,Invoices to be issued,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1510,1510,Outstanding receivables,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1511,1511,Outstanding bills of exchange,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1531,1531,Receivables to be settled,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1540,1540,Allowance for doubtful accounts,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1541,1541,Provision for credit risks,asset_receivable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
+1601,1601,VAT n/credit,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
```
Comment (epictete) [inline addons/l10n_it/data/account.account.template.csv:40]: Not sure what this means :man_shrugging: 
Would like to check with PGI to make sure we get it right :wink:

**Comment by malb-odoo** [comment]
TODO: Check if the pot and po file are updated

File: addons/l10n_it/data/account.account.template.csv [hunk 18]
```diff
@@ -1,185 +1,185 @@
...
-1214,1214,Fondo ammortamento attrezzature commerciali,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1215,1215,Fondo ammortamento macchine d'ufficio,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1216,1216,Fondo ammortamento arredamento,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1217,1217,Fondo ammortamento automezzi,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1218,1218,Fondo ammortamento imballaggi durevoli,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1220,1220,Fornitori immobilizzazioni c/acconti,asset_non_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1301,1301,Mutui attivi,asset_fixed,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_ATT
-1401,1401,Materie di consumo,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
-1404,1404,Merci,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
-1410,1410,Fornitori c/acconti,asset_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_ATT
```
Comment (epictete) [inline addons/l10n_it/data/account.account.template.csv:26]: Could you please translate this file again after removing the abbreviations like : c/, v/, n/...

File: addons/l10n_it/data/account.account.template.csv [hunk 19]
```diff
@@ -1,185 +1,185 @@
...
+2211,2211,Scheduled maintenance fund,liability_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_B_PASS
+2301,2301,TFRL debts,liability_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_C_PASS
+2410,2410,Mortgages payable,liability_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_D_PASS
+2411,2411,Banks grants,liability_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_D_PASS
+2420,2420,Banks in collection,liability_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_D_PASS
+2421,2421,Banks RIBA account in collection,liability_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_D_PASS
+2422,2422,Banks account bills in collection,liability_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_D_PASS
+2423,2423,Bank account advances on invoices,liability_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_D_PASS
+2440,2440,Debts to other lenders,liability_current,FALSE,l10n_it_chart_template_generic,l10n_it.account_tag_D_PASS
+2501,2501,Debts to suppliers,liability_payable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_D_PASS
```
Comment (epictete) [inline addons/l10n_it/data/account.account.template.csv:69]:
```suggestion
2501,2501,Accounts payable,liability_payable,TRUE,l10n_it_chart_template_generic,l10n_it.account_tag_D_PASS
```

**Review by epictete — COMMENTED** [review]
Tiny things

**Comment by epictete** [comment]
@robodoo r+

**Pull request closed without merging**